app/graphql/mutations/update_user.rb

```diff
@@ -47,6 +44,15 @@ def changing_password?(attributes)
       [:current_password, :password, :password_confirmation].any? { |k| attributes.include?(k) && attributes[k].present? }
     end
 
+    def prune_restricted_values(user, attributes)
+      attributes.delete(:is_admin) unless ability.can?(:manage, :all)
```

Comment:
```suggestion
      # Only administrators can make other users into administrators
      attributes.delete(:is_admin) unless ability.can?(:manage, :all)
```
Just a comment suggestion for clarity, since we have the other helpful comments below.